free-courses-en.md

```diff
@@ -371,6 +372,12 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Kotlin
+
+* [Android Kotlin Tutorial: Create Android Apps using Kotlin](https://www.youtube.com/watch?v=e7WIPwRd2s8&list=PLlxmoA0rQ-Lw5k_QCqVl3rsoJOnb_00UV) - Sriyank Siddhartha (Youtube)
```

Comment:
Could you please use a link from the playlist endpoint, instead of the watch endpoint?
Also, it'd be better if the noun "YouTube" was written with a capital T.

```suggestion
* [Android Kotlin Tutorial: Create Android Apps using Kotlin](https://www.youtube.com/playlist?list=PLlxmoA0rQ-Lw5k_QCqVl3rsoJOnb_00UV) - Sriyank Siddhartha (YouTube)
```

Reply:
Awesome - I was trying to find the proper landing page for youtube playlists - thanks!